src/lib.rs

```diff
@@ -528,6 +539,19 @@ fn handle_profile_override<C: VgpuConfigLike>(config: &mut C) -> bool {
     let vgpu_type = format!("nvidia-{}", config.vgpu_type());
     let mdev_uuid = LAST_MDEV_UUID.lock().take();
 
+    //output mdev uuid and vmid info
```

Comment:
These log statements are not necessary as there are log statements if the appropriate overrides are found.

Reply:
In my situation, the issue I was troubleshooting was that the overrides were no longer being hit.  It helped to know whether the wrong mdev was assigned or none at all (as it turned out).  Regardless, I'll clean it up.